### DISCUSSION TITLE
Hight-light of row selected in DataTable

### DISCUSSION THREAD

**Discussion by luongminh2111** [primefaces/community#1606] [PrimeReact] [answered]
When i updated to version 10.6.2, in DataTable, i can't see classname "p-hightlight" in selection mode single or multiple when i selecte row, please help me

**Comment by melloware** [comment, marked as answer]
Already reported: https://github.com/primefaces/primereact/issues/6290

Fixed for next release

**Reply by luongminh2111** [reply]
thanks
